pro/src/screens/OfferEducationalStock/FormStock/FormStock.tsx

```diff
@@ -14,25 +14,32 @@ import {
 
 import styles from './FormStock.module.scss'
 
-const EACOfferStockCreationType = (): JSX.Element => {
+const EACOfferStockCreationType = ({
+  isEditable,
+}: {
+  isEditable: boolean
+}): JSX.Element => {
   const { values } = useFormikContext<OfferEducationalStockFormValues>()
   return (
     <>
       <DatePicker
         className={styles['form-field']}
+        disabled={!isEditable}
```

Comment:
si on suit la préconisation plus haut de passer un mode read_only : 

```tsx
// plus haut 
const isFormDisabled = mode === Mode.READ_ONLY
```

```suggestion
        disabled={isFormDisabled}
```